recipes/ktx/all/conanfile.py

```diff
@@ -0,0 +1,137 @@
+from conans import ConanFile, CMake, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class KtxConan(ConanFile):
+    name = "ktx"
+    description = "Khronos Texture library and tool."
+    license = "Apache-2.0"
+    topics = ("conan", "ktx", "texture", "khronos")
+    homepage = "https://github.com/KhronosGroup/KTX-Software"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "sse": [True, False],
+        "tools": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "sse": True,
+        "tools": True,
+    }
+
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _has_sse_support(self):
+        return self.settings.arch in ["x86", "x86_64"]
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+        if not self._has_sse_support:
+            del self.options.sse
+        if self.settings.os in ["iOS", "Android", "Emscripten"]:
+            # tools are not build by default if iOS, Android or Emscripten
+            self.options.tools = False
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, 11)
+
+    def requirements(self):
+        self.requires("lodepng/cci.20200615")
+        self.requires("zstd/1.4.9")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def build(self):
+        self._patch_sources()
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def _patch_sources(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+        cmakelists = os.path.join(self._source_subfolder, "CMakeLists.txt")
+        # Allow CMake wrapper
+        tools.replace_in_file(cmakelists, "${CMAKE_SOURCE_DIR}", "${CMAKE_CURRENT_SOURCE_DIR}")
+        tools.replace_in_file(cmakelists, "${CMAKE_BINARY_DIR}", "${CMAKE_CURRENT_BINARY_DIR}")
+        # Unvendor several libs (we rely on CMake wrapper to link those libs)
+        # It's worth noting that vendored jpeg-compressor can't be replaced by CCI equivalent
+        basisu_dir = os.path.join(self.build_folder, self._source_subfolder, "lib", "basisu")
+        ## lodepng (the patch file 0002-lodepng-no-export-symbols is important, in order to not try to export lodepng symbols)
+        tools.remove(os.path.join(basisu_dir, "encoder", "lodepng.cpp"))
+        tools.remove(os.path.join(basisu_dir, "encoder", "lodepng.h"))
```

Comment:
```suggestion
        os.remove(os.path.join(basisu_dir, "encoder", "lodepng.cpp"))
        os.remove(os.path.join(basisu_dir, "encoder", "lodepng.h"))
```

revert please

Reply:
Why? 😕 I've been busy for a week, did I miss something?

Reply:
Because this method doesn't even exist, I don't understand why it was suggested.

Reply:
Ahh because I thought it was https://docs.conan.io/en/latest/reference/tools.html#tools-rename, silly mistake on my part

Thanks for catching it!